pygmt/src/velo.py

```diff
@@ -5,17 +5,24 @@
 import pandas as pd
 from pygmt.clib import Session
 from pygmt.exceptions import GMTInvalidInput
-from pygmt.helpers import build_arg_string, fmt_docstring, kwargs_to_strings, use_alias
+from pygmt.helpers import (
+    build_arg_string,
+    deprecate_parameter,
+    fmt_docstring,
+    kwargs_to_strings,
+    use_alias,
+)
 
 
 @fmt_docstring
+@deprecate_parameter("color", "fill", "v0.8.0", "v0.12.0")
 @use_alias(
     A="vector",
     B="frame",
     C="cmap",
     D="rescale",
     E="uncertaintycolor",
```

Comment:
I'm wondering if we should also deprecated `uncertaintycolor` to `uncertaintyfill`.

Reply:
In my opinion, yes!

Reply:
Done @seisman.